### ISSUE TITLE
Stage 0: a black sheep?

### ISSUE THREAD

**Issue by yaronf** [ietf-satp/draft-ietf-satp-architecture#11]
I don't see how we can ever have interoperability if Stage 0 is out of scope. We could define a minimal set of data elements being transmitted, and expect extensions to cover the legal requirements that will for sure pop up. There is no forward compatibility issue because gateways can always refuse incoming requests.

Also, version negotiation should be in Stage 0 and not Stage 1 (see Sec. 9).

**Comment by findthomas** [comment]
Agree.  We are early days in working on Stage-0 in that we have just started discussing Stage-0 in earnest in the past 3 months.

This was the focus of Denis' presentation at IETF117.

**Comment by findthomas** [comment]
Stage 0 is currently out of scope, and will be addressed hopefully after the re-chartering. 

Denis' draft-avrilionis-satp-setup-stage-01 already begins taking about this topic.
